release/ray_release/scripts/build_pipeline.py

```diff
@@ -151,6 +152,8 @@ def main(
             "Empty test collection. The selected frequency or filter did "
             "not return any tests to run. Adjust your filters."
         )
+    logger.info("Build anyscale BYOD images")
+    _build_anyscale_byod_images(filtered_tests)
```

Comment:
I'm not sure if this should happen in the pipeline builder script. It presumably takes quite a bit of time to build these images. Should this happen here? Are tests sharing the same image?

Reply:
Sorry for the lack of details. Hopefully this give more context: https://docs.google.com/document/d/12WU4vpR0jHgOwceoSgPtE4B-B-6mBonnHHiCSMmMNoY/edit#

To your question, this doesn't rebuild ray image, only anyscale-ray image so took about 10 minute to finish. But yes, I want to build the image once and reuse them across all tests (a nice side effect is that the customer environment build that take about 10 minute for each test will be 0).

Reply:
OK, thanks for the context. That sounds good to me